hi-ml-azure/src/health_azure/himl.py

```diff
@@ -193,8 +197,24 @@ def create_run_configuration(
     """
     run_config = RunConfiguration()
 
+    # If both docker_build_context and conda_environment_file are provided, throw an error
+    if docker_build_context and conda_environment_file:
+        raise ValueError("Both docker_build_context and conda_environment_file cannot be provided at the same time.")
+
     if aml_environment_name:
         run_config.environment = Environment.get(workspace, aml_environment_name)
+    elif docker_build_context:
+        # Check that the dockerfile exists
+        dockerfile_path = Path(docker_build_context.location) / docker_build_context.dockerfile_path
+        if not dockerfile_path.exists():
+            raise ValueError("Dockerfile not found in the provided docker_build_context.")
+        environment_name = load_and_hash_directory(Path(docker_build_context.location))
```

Comment:
Maybe do `docker_build_context_dir = Path(docker_build_context.location)` and then use that through the method?